packages/time-utilities/src/lib/DurationFormatter.ts

```diff
@@ -35,10 +35,10 @@ export class DurationFormatter {
 		if (negative) duration *= -1;
 
 		for (const [type, timeDuration] of kTimeDurations) {
-			const substraction = duration / timeDuration;
-			if (substraction < 1) continue;
+			const subtraction = duration / timeDuration;
```

Comment:
```suggestion
			const division = duration / timeDuration;
```

it's a division after all

Reply:
yeah idk where substraction came from lol